aspnet/identity/overview/features-api/customize_identity_model.md

```diff
@@ -775,9 +787,12 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
 ### Lazy loading
 
-Finally, let's add support for lazy-loading proxies in the Identity model. This can be very useful since it allows transparent navigation from on Identity entity to another related entity without first having to check whether the related entity has actually been loaded.
+In this section support for lazy-loading proxies in the Identity model is added. Lazy-loading proxies can be useful since it allows:
+
+* Transparent navigation from one Identity entity to another related entity.
```

Comment:
This makes it sound like it is two ways in which lazy-loading can be useful. It's just one: that navigation can be done without first ensuring the navigations have been loaded.

Reply:
Anything more than 25 words is difficult for native speakers and impossible for MT.

Reply:
How about: "Lazy-loading is useful since it allows navigation properties to be used without first ensuring they are loaded." Captures what I wanted to say and only 18 words! ;-)